Makefile

```diff
@@ -1,5 +1,5 @@
 TOP=.
-SUBDIRS=builds runtime fsharp src tools msbuild
```

Comment:
Why is this change of position important?

Reply:
It's important when building mlaunch from source, because mlaunch is a Xamarin.Mac project, and it needs the Xamarin.Mac msbuild targets / assemblies, which means that we need to build the msbuild directory before we build mlaunch (which is built in tools/mlaunch).